tests/pyret/tests/test-statistics.arr

```diff
@@ -49,18 +49,45 @@ check "numeric helpers":
   modes([list: -1, 2, -1, 2, -1]) is [list: -1]
   modes([list: 1, 1, 2, 2, 3, 3, 3]) is [list: 3]
 
+  variance([list:]) raises "empty"
+  variance([list: 5]) is 0
+  variance([list: 3, 4, 5, 6, 7]) is%(within(0.01)) 2.0
+  variance([list: 1, 1, 1, 1]) is-roughly ~0
+
   stdev([list:]) raises "empty"
   stdev([list: 5]) is 0
-
   stdev([list: 3, 4, 5, 6, 7]) is%(within(0.01)) 1.41
   stdev([list: 1, 1, 1, 1]) is-roughly ~0
-  stdev([list:]) raises "empty"
-  
+
+  variance-sample([list: 3, 4, 5, 6, 7]) is%(within(0.01)) (10 / 4)
+  variance-sample([list: 3]) raises "division by zero"
+  variance-sample([list: 1, 1, 1, 1]) is-roughly ~0
+  variance-sample([list:]) raises "empty"
+
   stdev-sample([list: 3, 4, 5, 6, 7]) is%(within(0.01)) num-sqrt(10 / 4)
   stdev-sample([list: 3]) raises "division by zero"
   stdev-sample([list: 1, 1, 1, 1]) is-roughly ~0
   stdev-sample([list:]) raises "empty"
 
+  z-test([list: 1, 2, 3], 1.58, 2.58) is%(within(0.01)) -0.6358
```

Comment:
Nice, thanks.  I'm curious why using `is%(within(0.01))` and not give a few more digits and just use `is-roughly`?  No need to change (unless it's easy!)

Reply:
Done³